.gitignore

```diff
@@ -9,10 +9,10 @@ pkg/
 # Berkshelf
 .vagrant
 /cookbooks
-# Berksfile.lock
+Berksfile.lock
```

Comment:
I think you're misreading this, he's telling gitignore to ignore that file, he's not adding the file